packages/components/src/navigation/index.js

```diff
@@ -30,6 +30,11 @@ export default function Navigation( {
 	const navigationTree = useCreateNavigationTree();
 
 	const setActiveMenu = ( menuId, slideInOrigin = 'left' ) => {
+		const menuExists = !! navigationTree.getMenu( menuId );
+		if ( ! menuExists ) {
+			return;
+		}
```

Comment:
Nitpick: this should be readable enough even as:
`if ( ! navigationTree.getMenu( menuId ) )`